code/game/objects/effects/decals/cleanable.dm

```diff
@@ -183,12 +183,6 @@ var/list/infected_cleanables = list()
 			perp.add_blood_to_feet(amount, basecolor, blood_DNA)
 			amount--
 
-/obj/effect/decal/cleanable/proc/messcheck(var/obj/effect/decal/cleanable/M)
```

Comment:
Why is this here??

Reply:
not really sure, probably legacy code

Reply:
No I mean why are you including this change in the PR? Don't remove things that aren't in-scope

Reply:
proc becomes unused after this, wasn't even doing anything anyways